cgroup/cgroup.go

```diff
@@ -76,26 +63,23 @@ func (m cgroup) generatePath(kind string) (string, error) {
 
 // Read reads the given cgroup file data and returns the content as a string
 func (m cgroup) Read(kind, file string) (string, error) {
```

Comment:
Can we rename `kind` here to `controller`? It makes it _way_ more understandable what we're doing to use the actual cgroup jargon, and not a term that has another meaning within the codebase. I realize this predates all your refactorings, but now is a good time for the switchover

Reply:
sure, i will update.